packages/dashboard/src/customization/propertiesSections/settings/section.tsx

```diff
@@ -43,17 +59,45 @@ export const SettingsSection = ({
             className='settings-property-label'
             style={{ gap: awsui.spaceScaledS }}
           >
-            <label htmlFor='decimal-places'>Decimal places</label>
-            <div className='settings-property-label-control'>
-              <Input
-                type='number'
-                data-testid='decimal-place-config'
-                controlId='decimal-places'
-                ariaLabel='decimal places'
-                value={significantDigits?.toFixed() ?? ''}
-                onChange={onSignificantDigitsChange}
-              />
-            </div>
+            <label htmlFor='decimal-places'>
+              <b>Places</b>
+            </label>
+            <Controller
+              control={control}
+              name='decimalPlaces'
+              rules={{
+                max: {
+                  value: 100,
```

Comment:
Please also include a rule to throw a validation error when less than zero.

Reply:
Done